recipe/meta.yaml

```diff
@@ -0,0 +1,131 @@
+  run_exports:
+    # Just guessing since the library is so new
+    # but they seem to want to be backward compatible
+    # https://abi-laboratory.pro/index.php?view=timeline&l=zlib-ng
+    - {{ pin_subpackage('zlib-ng', max_pin='x.x') }}  # [compat == "no"]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ stdlib('c') }}
+    - cmake
+    - make  # [unix]
+    - gtest  # [compat == "yes"]
+
+test:
+  commands:
+    # compat builds a standalone z library that does not depend on zlib-ng
+    - test -e $PREFIX/include/zlib-ng.h                         # [unix and compat == "no"]
+    - test -e $PREFIX/include/zconf-ng.h                        # [unix and compat == "no"]
+    - test -e $PREFIX/lib/libz-ng${SHLIB_EXT}                   # [unix and compat == "no"]
+    - test -e $PREFIX/lib/pkgconfig/zlib-ng.pc                  # [unix and compat == "no"]
+    - if not exist %LIBRARY_INC%\zlib-ng.h exit 1               # [win and compat == "no"]
+    - if not exist %LIBRARY_INC%\zconf-ng.h exit 1              # [win and compat == "no"]
+    - if not exist %LIBRARY_BIN%\zlib-ng2.dll exit 1            # [win and compat == "no"]
+    - if not exist %LIBRARY_LIB%\zlib-ng.lib exit 1.            # [win and compat == "no"]
+    - if not exist %LIBRARY_LIB%\pkgconfig\zlib-ng.pc exit 1    # [win and compat == "no"]
+
+outputs:
+  # zlib-ng gets built when compat is NOT enabled; versioned with the regular zlib-ng release
+  - name: zlib-ng
+    build:
+      skip: true  # [compat == "yes"]
+  # zlib and libzlib get built when compat is enabled; they are versioned after the zlib API
+  - name: libzlib
+    version: {{ zlib_api }}
+    build:
+      skip: true  # [compat == "no"]
+      string: h{{ PKG_HASH }}_ng_{{ version.replace(".", "") }}_{{ number }}
+      run_exports:
+        # This is for compat mode, just libzlib, with major pinning
+        - {{ pin_subpackage('libzlib', max_pin='x') }}
+      track_features:
+        - zlib-ng-compat
+    files:
+      include:
+        - lib/libz.so.*          # [linux]
+        - lib/libz.*.dylib       # [osx]
+        - Library/bin/zlib.dll   # [win]
+        - zlib.dll               # [win]
+    test:
+      commands:
+        - test ! -e $PREFIX/include/zlib.h                          # [unix]
+        - test ! -e $PREFIX/lib/libz.a                              # [unix]
+        - test ! -e $PREFIX/lib/libz${SHLIB_EXT}                    # [unix]
+        - if not exist %LIBRARY_BIN%\zlib.dll exit 1                # [win]
+        - if not exist %PREFIX%\zlib.dll exit 1                     # [win]
+  - name: zlib
+    version: {{ zlib_api }}
+    build:
+      skip: true  # [compat == "no"]
+      number: {{ number }}
+      string: h{{ PKG_HASH }}_ng_{{ version.replace(".", "") }}_{{ number }}
+      run_exports:
+        - {{ pin_subpackage('libzlib', max_pin='x') }}
+        - {{ pin_subpackage('zlib-ng-compat', exact=True) }}
```

Comment:
```suggestion
```

This goes in the `libzlib` run dependencies.

Reply:
Sorry, I thought I've merged it.